src/ros_filter.cpp

```diff
@@ -430,6 +448,119 @@ namespace RobotLocalization
   }
 
   template<typename T>
+  void RosFilter<T>::publishState(const ros::Time)
+  {
+      // Get latest state and publish it
+      nav_msgs::Odometry filteredPosition;
```

Comment:
pedantic: same as my first comment: `filteredPosition` might be confusing, it should be `filteredOdometry`